build.sbt

```
@@ -18,6 +18,25 @@ inThisBuild(
   )
 )
 
+lazy val root = project
+  .in(file("."))
+  .settings(
+    // for scala-steward
```

```suggestion
    // for scala-steward
    publish / skip := true,
    // silent the intransitive dependency warning
    publishMavenStyle := false,
```
maybe? We do it in Metals https://github.com/scalameta/metals/blob/main/build.sbt#L771